CHANGELOG.md

```diff
@@ -1,3 +1,123 @@
+# 9.1.0 "adamantium-mannequin" (2020-02-24)
+
+### Highlights
+
+- Column resizing for tables is now in experimental ([198911](https://github.com/angular/components/commit/198911))
+- Selection list now supports a single selection mode. Enable it with the `multiple` input set to `false`.
+- `FakeMatIconRegistry` has been added for unit testing.
```

Comment:
- Added a new Sass mixin, `mat-strong-focus-indicators`, that can be used to add a stronger focus indicator to Angular Material components